### ISSUE TITLE
Controller only works when swapping to another port.

### ISSUE THREAD

**Issue by Marotheit** [mgba-emu/mgba#908]
**_I am using the Mayflash WiiU GameCube Adapter in PC Mode with a stock GameCube controller plugged into port 1._**

mGBA only seems to recognize inputs after the "correct port" is selected from the drop-down menu of the controller menu in the settings. If the emulator is closed, the process must be repeated when the emulator is re-opened to regain control of the emulator through a controller.

I say correct port because the controller is plugged into port 1, and recognized as such in every program I've configured it for, but mGBA doesn't seem to agree.

To attempt to disprove user error (although I recognize that may very well be the case,) I have included my configuration here: [https://pastebin.com/6gTHiAGq](https://pastebin.com/6gTHiAGq)

Edit: I am running the latest version of mGBA on Windows 10.

**Comment by endrift** [comment]
Yikes, that config file is corrupted. I wonder why it's getting that way.

**Comment by Marotheit** [comment]
I've actually reset and re-generated the config file multiple times, and it always ends up that way.

**Comment by endrift** [comment]
Yup, I think I've found the source of the corruption. That said, I'm not sure if that will fix all of the issues you're having.

**Comment by Marotheit** [comment]
Even if it doesn't, I'm glad to contribute! Once you have it fixed, let me know and I'll compile the build and see if it fixed it. Either way, I appreciate the support!

**Comment by JMC47** [comment]
Was this ever addressed?  I have a Mayflash GameCube Adapter and can test this if need be.

**Comment by Marotheit** [comment]
I did not experience the issue after the next version was released, so safe
to mark it solved as far as I was concerned.

**Comment by cinnamonrollls** [comment]
Same issue in version 0.10.2
[Mayflash Bug.zip](https://github.com/mgba-emu/mgba/files/13200275/Mayflash.Bug.zip)
